_layouts/product.html

```diff
@@ -91,8 +91,13 @@
 				<section class="product-info">
 					<h2>{{ product.name }}</h2>
 					<h4>
-						<a href="http://{{ product.url }}"target="blank">
-							{{ product.url }}
+						{% if product.url contains "http://" or product.url contains "https://" %}
```

Comment:
I wanted a regex matcher here, or at least something to emulate "starts with", but it doesn't appear that Liquid has anything like this.